package.json

```diff
@@ -14,45 +14,47 @@
   },
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
     "test": "ember try:each",
     "deploy": "ember github-pages:commit --message \"Deploy gh-pages from commit $(git rev-parse HEAD)\"; git push; git checkout -"
   },
   "dependencies": {
-    "broccoli-flatiron": "0.0.0",
+    "broccoli-flatiron": "0.1.2",
     "broccoli-merge-trees": "^2.0.0",
     "broccoli-writer": "^0.1.1",
     "ember-cli-babel": "^6.6.0",
     "ember-cli-htmlbars": "^2.0.3",
-    "ember-cli-sass": "^6.1.3",
+    "ember-cli-sass": "^7.1.4",
```

Comment:
No need for any change in this PR... just calling this out for posterity:

Some folks have been having problems with ember-cli-sass dependency resolution based on their local app's setup. I believe ember-cli-sass 7 (released in July) has some breaking changes from ember-cli-sass 6, so I don't know if this upgrade will help or hurt. Regardless, I'm going to change ember-freestyle's scss to vanilla CSS soon, so I say we roll with this as is for the time being.